lib/broccoli/ember-app.js

```diff
@@ -945,7 +945,7 @@ class EmberApp {
 
   _nodeModuleTrees() {
     if (!this._cachedNodeModuleTrees) {
-      this._cachedNodeModuleTrees = this._nodeModules.map(module => new Funnel(module.path, {
+      this._cachedNodeModuleTrees = Array.from(this._nodeModules.values(), module => new Funnel(module.path, {
```

Comment:
@rwjblue Looked at broccoli-funnel-reducer as you'd suggested, but that seems like it's only helpful if you have a bunch of funnels sharing an input node, which is exactly what we're trying to avoid here :)